openshift/scripts/oc_cleanup_test.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh -l
```

Comment:
See comment above re. only having one cleanup script, and specifying the target.